packages/react-components/react-charts-preview/library/src/components/VerticalBarChart/useVerticalBarChartStyles.styles.ts

```diff
@@ -64,10 +52,7 @@ export const useVerticalBarChartStyles_unstable = (props: IVerticalBarChartProps
   const baseStyles = useStyles();
 
   return {
-    opacityChangeOnHover: mergeClasses(
-      verticalbarchartClassNames.opacityChangeOnHover,
-      baseStyles.opacityChangeOnHover /*props.styles?.opacityChangeOnHover*/,
-    ),
+    opacityChangeOnHover: mergeClasses(verticalbarchartClassNames.opacityChangeOnHover),
```

Comment:
why this change

Reply:
it was giving build error as opacityChangeOnHover is not present in baseStyles